docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -262,6 +262,11 @@ that import the namespaced Kubernetes resources (kinds of objects) for managemen
 containerized workloads. At the completion of `kubestellar init` the current workspace will be
 "root".
 
+6. Creates the space provider objects to be used by the space manager.  Need to pass 
+in the kubeconfig for the space manager using SPACE_MANAGER_KUBECONFIG env variable, as
+well as specify using the IN_CLUSTER environment variable whether the spaces are accessed 
```

Comment:
Does the `IN_CLUSTER` environment variable talk indicate whether the `kubestellar init` command is being invoked inside the hosting cluster? If so then it needs to be described that way rather than as "whether the spaces are accessed from within the cluster or externally"; speaking of "are accessed" without identifying _by what_ leaves the reader lost and confused.